app.py

```diff
@@ -0,0 +1,52 @@
+from flask import Flask, redirect, render_template, request
+from flask_cors import CORS 
+from flask_sqlalchemy import SQLAlchemy
+from controllers import urls as Urls
+
+
+#from werkzeug import exceptions
+
+app = Flask(__name__)
+CORS(app)
+
+POSTGRES = {
+    'user': 'debmon',
+    'pw': 'secretpassword',
```

Comment:
secretpassword!! maybe store in an environment variable, you could use the os module:  https://able.bio/rhett/how-to-set-and-get-environment-variables-in-python--274rgt5